pypeit/pypeit.py

```diff
@@ -798,20 +757,18 @@ def objfind_one(self, frames, det, bg_frames, std_outfile=None):
                                                    ignore_saturation=False)
             sciImg = sciImg.sub(bgimg)
 
-        # Check if the user has manually created a Master sky regions
-        sky_region_file = None
-        if self.par['reduce']['skysub']['user_regions'] == 'master':
-            # Check if a master Sky Regions file exists for this science frame
-            file_base = os.path.basename(sciImg.files[0])
-            prefix = os.path.splitext(file_base)
-            if prefix[1] == ".gz":
-                sciName = os.path.splitext(prefix[0])[0]
-            else:
-                sciName = prefix[0]
-
-            master_dir = self.caliBrate.master_dir
-            master_key = self.caliBrate.fitstbl.master_key(0, det=self.det) + "_" + sciName
-            sky_region_file = masterframe.construct_file_name(buildimage.SkyRegions, master_key, master_dir=master_dir)
+        # TODO: The spatial flexure is not copied to the PypeItImage object if
```

Comment:
@jhennawi should comment here

Reply:
I'm not completely sure I follow. But if the flexure correction is performed on a combined image which seems to be the only sensible thing you could do, I think the information should be propagated.

Reply:
My comment wasn't very well written.  The issue was that, when background images were being subtracted from the main science image, the `PypeItImage.sub` function was not propagating any spatial flexure measurement to the subtracted image.  I've now changed it so it is propagated, and I added this to the doc string of the sub function:
```
            - if the spatial flexure is defined for the first image, it is
              propagated regardless of the value for the 2nd image.  If it is
              also defined for the 2nd image and the flexure is different from
              the first image, a warning is issued.  If the flexure is only
              defined for the 2nd image, it is ignored.
```